elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/MetaDataStore.java

```diff
@@ -32,28 +39,52 @@
     public static final Package META_DATA_PACKAGE =
             Package.getPackage("com.yahoo.elide.datastores.aggregation.metadata.models");
 
-    public MetaDataStore() {
+    private static final Class[] METADATA_STORE_CLASSES = {
+            FromTable.class, FromSubquery.class, Subselect.class, javax.persistence.Table.class};
+
+    /**
+     * A separate dictionary (not exposed to Elide core), that tracks all of the entities
+     * that the QueryEngine interacts with.
+     */
+    @Getter
+    EntityDictionary metadataDictionary;
+
+    @Getter
+    private final String scanPackageName;
+
+    public MetaDataStore(Package scanPackage) {
+        this (scanPackage.getName());
+    }
+
+    public MetaDataStore(String scanPackageName) {
         super(META_DATA_PACKAGE);
+        this.scanPackageName = scanPackageName;
```

Comment:
How does this handle the case that one metadatastore needs to manage more than one package?

Reply:
I'm going to punt on this one for now.  I had a similar thought, but likely a single package is sufficient for nearly every case.